src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int MAX_RANDOM_NUMBER = 10;
+    public static final int DEFAULT_RADIUS = 10;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+        String color = colorSupplier.getRandomColor();
+        int radius = random.nextInt(MAX_RANDOM_NUMBER);
+        Figure randomFigure = new Circle(color, radius);
```

Comment:
let's return random circle in case if index is 0

Reply:
When i dont initialize Figure randomFigure at start of method getRandomFigure, and put all initilizations in cases, then IDEA say:
Variable 'randomFigure' might not have been initialized
How can i solve this problem?
Initialize with null?
I maked it but no sure that is right choice.